### DISCUSSION TITLE
API INVOICE - Come ricevere il numero dell'ultima fattura emessa

### DISCUSSION THREAD

**Discussion by Mustino2022** [fattureincloud/api#319] [Q&A] [answered]
Buongiorno,
come è possibile ricevere tramite le API il numero dell'ultima fattura emessa?
Al momento utilizzo la Get per ricevere la lista dei documenti emessi, filtrati per:
type=invoice
per_page=100
ListIssuedDocumentsResponse resultDocument = apiInstance.ListIssuedDocuments((int)IDCompany, "invoice",null,null,"id",null,100,"",0);

Non sono sicuro che sia il modo migliore per ricevere il numero dell'ultima fattura emessa

**Comment by valmoz** [comment, marked as answer]
Buongiorno @Mustino2022 ,
le suggerisco di usare l'ordinamento: [Sort Results](https://developers.fattureincloud.it/docs/basics/additional-capabilities/sort-results/)

Inoltre, se le serve solo l'ultimo numero di fattura, non è molto utile utilizzare per_page a 100.

Ci faccia sapere se questo può esserle utile.

Grazie
Mauro

**Reply by Mustino2022** [reply]
Buongiorno @valmoz 
grazie per il suggerimento, funziona correttamente.
Buona giornata
